File: Demohahaha/VanillaSyrup.cs
public class VanillaSyrup : ToppingDecorator{

    public VanillaSyrup(EspressoBeverage newEspBev) : base(newEspBev){
        Console.WriteLine("Adding Vanilla Syrup");

    }
    new public string getDescription(){
        return tempEspresso.getDescription() + ", Vanilla Syrup";
    }
    new public double getCost(){
        return tempEspresso.getCost() + 0.80;
    }
}

File: src/main/java/VanillaSyrup.java
public class VanillaSyrup extends ToppingDecorator{

    public VanillaSyrup(EspressoBeverage newEspBev) {
        super(newEspBev);
        System.out.println("Adding Vanilla Syrup");

    }
    public String getDescription(){
        return tempEspresso.getDescription() + ", Vanilla Syrup";
    }
    public double getCost(){
        return tempEspresso.getCost() + 0.80;
    }
}
